loader/loader.go

```diff
@@ -358,6 +358,12 @@ func loadYamlModel(ctx context.Context, config types.ConfigDetails, opts *Option
 		dict = map[string]interface{}{}
 		err  error
 	)
+	var interpolateOptsBeforeMerge interp.Options
+
+	if opts.Interpolate != nil && !opts.SkipInterpolation {
+		interpolateOptsBeforeMerge = opts.Interpolate.Clone()
+		interpolateOptsBeforeMerge.KeysToInterpolate = []string{"include", "extends"}
```

Comment:
AFAICT this would interpolate `include` but will skip `include.path`, won't it?
This deserves a dedicated test as there's a high risk to introduce a regression with such a change
I also think it would be safer to filter by path (`services.*.extends.**`)  vs attribute name

Reply:
Yes, it will. Fixed it, now it checks if the Path contains a given part.  
Before writing tests I want to know do we agree to do it this way?